projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts

```diff
@@ -27,6 +29,13 @@ interface IAngularExtensionConfig {
     errorServices?: IErrorService[];
 }
 
+let undefValue = undefined;
+
+const defaultAngularExtensionConfig: IConfigDefaults<IAngularExtensionConfig> = objDeepFreeze({
+    router: undefValue,
```

Comment:
We should use the `blkVal` definition for this as well as the router (might) be an object.